### ISSUE TITLE
Recurrence MVP: Basic `startRecurrence` mutation + recurrence objects

### ISSUE THREAD

**Issue by jmtaber129** [ParabolInc/parabol#6916]
Backend spec: https://github.com/ParabolInc/parabol/issues/6888

Implement the `startRecurrence` mutation, along with any additional recurrence objects (e.g. meeting series) necessary:
* Create the new `MeetingSeries` object in postgres + graphql
* Add the new fields to the `NewMeeting` rethink object
* Create the `startRecurrence` mutation, which takes a meeting ID as an argument

Out of scope: 
* Anything on the frontend
* Restarting recurrence

The intention is to implement the basic foundation needed to execute other recurrence tasks (UI, `stopRecurrence`, `processRecurrence`, etc.) in parallel

### Acceptance Criteria
- [ ] Devs can run the `startRecurrence` mutation for a meeting in the graphql console, which results in the following
  - [ ] A new meeting series with relevant defaults exists
  - [ ] The meeting has a `scheduledEndTime` at a reasonable time in the future
  - [ ] The meeting has a `meetingSeriesId` associated with it in its rethink row
  - [ ] The recurrence rule represents a start time for the next meeting that makes sense.  E.g. the rrule shouldn't represent that the next meeting should start later that same day.
- [ ] Errors are returned if:
  - [ ] The meeting doesn't exist
  - [ ] The meeting already has a `meetingSeriesId` associated with it, and the series is inactive (restarting recurrence is out of scope)
